.github/workflows/ios-release-to-org-openfoodfacts-scanner.yml

```diff
@@ -46,6 +46,7 @@ jobs:
 
       - name: Get dependencies
         run: ci/pub_upgrade.sh
+        # or perhaps in that file ?
```

Comment:
```suggestion
```

Reply:
# or perhaps in that file ? https://github.com/openfoodfacts/smooth-app/blob/develop/ci/pub_upgrade.sh